nano/core_test/bootstrap_ascending.cpp

```diff
@@ -135,8 +135,13 @@ TEST (account_sets, priority_down_sat)
 	auto store = nano::make_store (system.logger, nano::unique_path (), nano::dev::constants);
 	ASSERT_FALSE (store->init_error ());
 	nano::bootstrap_ascending::account_sets sets{ system.stats };
-	sets.priority_down (account);
-	ASSERT_EQ (1.0f, sets.priority (account));
+	sets.priority_up (account);
+	ASSERT_GT (sets.priority (account), 1.0f);
+	for (int n = 0; n < 1000; ++n)
+	{
+		sets.priority_down (account);
+	}
+	ASSERT_EQ (0.0f, sets.priority (account));
```

Comment:
In addition to ASSERT_EQ on sets.priority we could check sets.priority_down so we'd have all the return values covered.